Feature/advance docs (#35)
infinum/eightshift-docs: docs/advanced/components-heading-level.md

---
id: components-heading-level
title: Heading Level
---

> This is a custom react component the renders a custom heading level. It is used in the heading component or block.

Reviewer: ```suggestion
This is a custom React component that renders a custom heading level (`<h1>` to `<h6>`). It is used in the heading component or block.
```

## Usage

```js
import { HeadingLevel } from 'EightshiftComponentHeadingLevel';

{onChangeLevel &&
  <HeadingLevel
    selectedLevel={level}
    onChange={onChangeLevel}
  />
}
```

### Options

#### minLevel

> Defines minimal value to show for heading 

Reviewer: ```suggestion
Defines the minimal value to show in a heading 
```

Default: 1

#### maxLevel

> Defines maximal value to show for heading 

Reviewer: ```suggestion
Defines the maximum value to show in a heading 
```

Default: 6

#### selectedLevel

> Defines selected level value to show for heading. 

Reviewer: ```suggestion
Defines the selected level value to show in a heading. 
```
